rts/at/util/src/main/java/org/jboss/jbossts/star/util/logging/RESTATLogger.java

```diff
@@ -0,0 +1,16 @@
+package org.jboss.jbossts.star.util.logging;
+
+
+	import org.jboss.logging.Logger;
+
+
+
+	public final class RESTATLogger {
+	    private RESTATLogger() {
+	    }
+
+	    public static final Logger logger = Logger.getLogger("org.jboss.jbossts.star.util.logging");
```

Comment:
I think the category should be here just `org.jboss.jbossts.star`. Other Narayana loggers do not use any suffix like here `util.logging`, plus I don't think it makes sense to say all log messages are util ones.

Reply:
And the name of the variable `at18NLogger` would be probably good to be renamed to `atI18NLogger` or maybe just to `i18NLogger` as using variable with name `i18NLogger` is a usual in other Narayana logger classes.

Reply:
+1 to both comments

Reply:
Changed naming